applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/volume/bt_vol_ctlr.c

```diff
@@ -10,14 +10,22 @@
 #include <zephyr/types.h>
 #include <zephyr/bluetooth/conn.h>
 #include <zephyr/bluetooth/audio/vcp.h>
+#include <zephyr/bluetooth/gatt.h>
+#include <../../../../subsys/bluetooth/audio/vcp_internal.h>
 
 #include "macros_common.h"
 #include "bt_rendering_and_capture.h"
 
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(bt_vol_ctlr, CONFIG_BT_VOL_LOG_LEVEL);
 
-static struct bt_vcp_vol_ctlr *vcs_client_peer[CONFIG_BT_MAX_CONN];
+struct vcs_peer {
+	struct bt_vcp_vol_ctlr *peer;
+	uint8_t master_vol;
+	bool default_val;
```

Comment:
Looking at the implementation, it appears that `default_val` indicates "unknown" value, because it's only `true` when you haven't stored a value, right?

Reply:
You are right. I have renamed.